sound/soc/sof/intel/Kconfig

```diff
@@ -61,6 +61,15 @@ config SND_SOC_SOF_INTEL_DISABLE_ATOM_SST_MERRIFIELD
 	  and UCM files.
 	  If unsure select N.
 
+config SND_SOC_SOF_INTEL_BUILDTIME_DISABLE_SKL_SST_DRIVER
+	bool "Build-time exclusion with the Skylake/SST driver"
+	help
+	  This option relies on build-time mutual exclusion with the
+	  Skylake/SST driver to select SOF. The SOF probe is not
+	  guaranteed to work if userspace does not provide a firmware
+	  signed with the relevant authentication key.
```

Comment:
Doesn't the same apply to SST as well  (probe will not work if correct firmware is available)? Oh well, I guess this is still useful for the distro audience, so I'm ok for leaving as is.

Reply:
yes, it's the same for the SKL/SST driver. We can't check for the firmware since it's done in a seconfd part after the probe completes. I will reword this, thanks for the feedback.